### PR TITLE
Upgrade to Hyper 1

### PR DESCRIPTION
**Pull request by sfackler**
Also bump rustls.

### CODE REVIEW

File: conjure-runtime/src/blocking/shim.rs [hunk 1]
```diff
@@ -1,212 +0,0 @@
-// Copyright 2020 Palantir Technologies, Inc.
```
Comment (sfackler) [inline conjure-runtime/src/blocking/shim.rs:1]: This was just dead code

File: conjure-runtime/src/body.rs [hunk 2]
```diff
@@ -139,13 +138,25 @@ where
...
+                .map_err(Error::internal_safe)?
+            {
+                Some(frame) => {
+                    if let Ok(data) = frame.into_data() {
+                        return Poll::Ready(Some(Ok(data)));
+                    }
+                }
+                None => return Poll::Ready(None),
+            }
+        }
```
Comment (pierceforte) [inline conjure-runtime/src/body.rs:159]: Should we have some delay in this loop?

File: conjure-runtime/src/body.rs [hunk 3]
```diff
@@ -212,32 +227,37 @@ where
...
+        Poll::Ready(frame)
     }
 
-    fn size_hint(&self) -> http_body::SizeHint {
-        self.body.size_hint()
+    fn is_end_stream(&self) -> bool {
+        if self.done {
+            true
+        } else {
+            self.body.is_end_stream()
```
Comment (pierceforte) [inline conjure-runtime/src/body.rs]: nit: `self.done || self.body.is_end_stream()`

**Review by pierceforte — DISMISSED** [review]
Approving with a couple comments

File: conjure-runtime/src/body.rs (hunk 2 above)
Comment (sfackler) [reply conjure-runtime/src/body.rs:159]: The poll_frame is going out to the network so we should be good.

File: conjure-runtime/src/body.rs (hunk 3 above)
Comment (sfackler) [reply conjure-runtime/src/body.rs]: Fixed!

**Comment by sfackler** [comment]
Needs a re-+1

**Review by pierceforte — APPROVED** [review]

**Pull request merged**